openff/toolkit/tests/test_molecule.py

```diff
@@ -4027,3 +4020,78 @@ def test_molecule_subclass_from_dict(self):
         orig_mol = Molecule.from_smiles("CCO")
         mol = MyMol.from_dict(orig_mol.to_dict())
         assert isinstance(mol, MyMol)
+
+
+class TestHierarchies:
+    def test_nothing_perceived_dipeptide(self, dipeptide):
+        with pytest.raises(KeyError) as context:
```

Comment:
```suggestion
        with pytest.raises(KeyError):
```
FYI: When using this context manager, you don't need to store the return value if not using it. Flake8 will complain about this